build/ci_utils/src/actions/artifacts.rs

```diff
@@ -12,7 +12,6 @@ use serde::de::DeserializeOwned;
 use tempfile::tempdir;
 
 
-
```

Comment:
Why do we remove this line?

Reply:
This was introduced by running `./run fmt` so I'd opt to leave it in.